src/com/javarush/task/task08/task0827/Solution.java

```diff
@@ -0,0 +1,44 @@
+package com.javarush.task.task08.task0827;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+/*
+Работа с датой
+*/
+
+public class Solution {
+    public static void main(String[] args) throws ParseException {
+        System.out.println(isDateOdd("MAY 1 2013"));
+    }
+
+    public static boolean isDateOdd(String date) throws ParseException {
+        SimpleDateFormat format = new SimpleDateFormat("MMMM d yyyy", Locale.ENGLISH);
+        Date dataDate = format.parse(date);
+        Calendar calendar = new GregorianCalendar();
+        calendar.setTime(dataDate);
+        int dayOfYear = calendar.get(GregorianCalendar.DAY_OF_YEAR);
+
+        return dayOfYear % 2 == 0;
```

Comment:
Неплохое решение, разве что format не обязателен можно было Date.parse(date)
и еще вот вариант:
```suggestion
        Date dateCheck = new Date(Date.parse(date));
        Date startOfTheYear = new Date(dateCheck.getYear(), 0, 0);
        long days = (dateCheck.getTime() - startOfTheYear.getTime()) / 1000 / 60 / 60 / 24;
        return days % 2 > 0;
```